aws/policy/security-services.yaml

```diff
@@ -27,6 +27,40 @@ Statement:
     Resource:
       - 'arn:aws:iam::{{ aws_account_id }}:role/ansible_lambda_role'
 
+  - Sid: AllowRegionalRestrictedResourceActionsWhichIncurFees
+    Effect: Allow
+    Action:
+      - wafv2:ListRuleGroups
+      - wafv2:ListWebACLs
+      - wafv2:AssociateWebACL
+      - wafv2:DeleteRuleGroup
+      - wafv2:CreateRuleGroup
+      - wafv2:PutFirewallManagerRuleGroups
+      - wafv2:GetWebACLForResource
+      - wafv2:GetLoggingConfiguration
+      - wafv2:DeleteWebACL
+      - wafv2:GetRateBasedStatementManagedKeys
+      - wafv2:ListLoggingConfigurations
+      - wafv2:GetIPSet
+      - wafv2:CreateWebACL
+      - wafv2:ListIPSets
+      - wafv2:GetWebACL
+      - wafv2:GetRuleGroup
+      - wafv2:CreateIPSet
+      - wafv2:ListAvailableManagedRuleGroups
+      - wafv2:DeleteIPSet
+      - wafv2:DescribeManagedRuleGroup
+      - wafv2:CheckCapacity
+      - wafv2:ListResourcesForWebACL
+      - wafv2:DeleteLoggingConfiguration
+      - wafv2:PutLoggingConfiguration
+      - wafv2:DisassociateWebACL
+      - wafv2:UpdateWebACL
+      - wafv2:UpdateRuleGroup
+      - wafv2:DeleteFirewallManagerRuleGroups
+      - wafv2:DisassociateFirewallManager
+      - wafv2:UpdateIPSet
```

Comment:
This needs a Resource to act on.  Looking at https://docs.aws.amazon.com/service-authorization/latest/reference/list_awswafv2.html#awswafv2-resources-for-iam-policies we can probably be pretty broad with this:
```
+    Resource:
+      - 'arn:aws:wafv2:{{ aws_region }}:{{ aws_account_id }}:*'
```

We're also missing a few policies, at least wafv2:TagResource
I also ran into this with 450: `not authorized to perform: elasticloadbalancing:SetWebACL on resource: arn:aws:elasticloadbalancing:us-east-1:966509639900:loadbalancer/app/my-alb-65921286/ad815a1d53f68410`